src/components/molecules/Settings/Workspace/MemberListItem/index.tsx

```diff
@@ -1,17 +1,20 @@
 import React, { useCallback } from "react";
 import { useIntl } from "react-intl";
+import { useMedia } from "react-use";
 
 import Flex from "@reearth/components/atoms/Flex";
 import Icon from "@reearth/components/atoms/Icon";
+import Text from "@reearth/components/atoms/Text";
 import Avatar from "@reearth/components/molecules/Settings/Avatar";
-import { metricsSizes, styled } from "@reearth/theme";
+import { metricsSizes, styled, useTheme } from "@reearth/theme";
 
 import EditableItem from "../../Project/EditableItem";
 
 export type Role = "READER" | "WRITER" | "OWNER";
 
 export type Props = {
   name?: string;
+  email?: string;
   role: Role;
```

Comment:
Instead of passing all three, can you condense this and just pass the whole user? The User does have id (which we won't use, and doesn't have role), but if you make it here:
```
user?: {
   name?: string;
   email?: string;
   role?: Role;
}
```
then I think you can pass it in as `user={...user, role}`.
What do you think?